test/verify/check-system-terminal

```diff
@@ -263,6 +264,98 @@ PROMPT_COMMAND='printf "\033]0;%s@%s:%s\007" "${USER}" "${HOSTNAME%%.*}" "${PWD/
         b.input_text("id\n")
         b.wait_js_cond("ph_text('.terminal').indexOf('uid=') >= 0")
 
+    @testlib.nondestructive
+    def testOpenPath(self):
+        b = self.browser
+        m = self.machine
+        b.default_user = "admin"
+
+        self.login_and_go("/system/terminal")
+
+        blank_state = ' '
+
+        # wait until first line is not empty
+        n = 1
+        b.wait_visible(".terminal .xterm-accessibility-tree")
+        function_str = "(function (sel) { return ph_text(sel).trim() != '%s'})" % blank_state
+        b.wait_js_func(function_str, line_sel(n))
+
+        # clear any messages (for example, instructions about sudo) and wait for prompt
+        b.input_text("clear")
+        b.wait_js_cond("ph_text('.terminal').indexOf('clear') >= 0")
+        # now wait for clear to take effect
+        b.key("Enter")
+        b.wait_js_cond("ph_text('.xterm-accessibility-tree').indexOf('clear') < 0")
+        # now we should get a clean prompt
+        b.wait_in_text(line_sel(n), '$')
+
+        # cut trailing non-breaking spaces
+        prompt = b.text(line_sel(n))
+
+        # Make sure we are started in home directory
+        # Account for non-standard prompting
+        if "]" not in prompt:
+            self.assertIn(":~$", prompt)
+        else:
+            self.assertIn("~]$", prompt)
+
+        # Change to directory that exists
+        b.go("#/?path=/tmp")
+        b.wait_visible(".terminal .xterm-accessibility-tree")
+        self.wait_line(n, "disconnected")
+        b.input_text("clear")
+        b.key("Enter")
+        b.wait_js_cond("ph_text('.xterm-accessibility-tree').indexOf('clear') < 0")
+        b.input_text("pwd\n")
+        self.wait_line(n + 1, "/tmp")
+
+        # Error on dir non-exist
+        b.go("#/?path=/doesnotexist")
+        b.wait_visible(".pf-v5-c-alert.pf-m-danger")
+        b.wait_in_text(".pf-v5-c-alert.pf-m-danger", "does not exist")
+        # If there's an error, we shouldn't reset the terminal
+        b.input_text("pwd\n")
+        self.wait_line(n + 3, "/tmp")
+        # Clear the error
+        b.click('.pf-v5-c-alert__action button')
+        b.wait_not_present(".pf-v5-c-alert.pf-m-danger")
+
+        # Error on change to non-dir
+        m.execute("touch /tmp/thisisafile")
+        b.go("#/?path=/tmp%2Fthisisafile")
+        b.wait_visible(".pf-v5-c-alert.pf-m-danger")
+        b.wait_in_text(".pf-v5-c-alert.pf-m-danger", "not a directory")
+        # If there's an error, we shouldn't reset the terminal
+        b.input_text("pwd\n")
+        self.wait_line(n + 5, "/tmp")
+        # Clear the error
+        b.click('div.pf-v5-c-alert__action button')
+        b.wait_not_present(".pf-v5-c-alert.pf-m-danger")
+        b.wait_js_cond('window.location.hash == "#/"')
+
+        # Prompt for change if terminal busy
+        b.input_text("vi busybusybusy\n")
```

Comment:
I'm not sure if all our images have `vi` (I only triggered on fedora-40 so far). Let's just use "sleep 100"?